src/styles/main.scss

```diff
@@ -1,7 +1,23 @@
-@import "utils";
-@import "fonts";
-@import "typography";
+@import "./utils/mixins.scss";
```

Comment:
Please import the 'typography' file to set the font styles and sizes according to the design.